docs/reference/developer_documentation.md

```diff
@@ -105,3 +105,20 @@ poetry publish --build
 1. [Download Node.js](https://nodejs.org/en/download/)
 2. [Install Angular](https://angular.io/guide/setup-local)
 3. See [buildingmotif-app/README.md](buildingmotif-app/README.md)
+
+
+## Running on Docker
+1. Set config file.
+    ```
+    cp configs.py.dist configs.py
```

Comment:
`configs.py.dist` doesn't exist. This line is obviated by the second line (`echo ... > configs.py`) so we should probably remove this line